src/init.cpp

```diff
@@ -956,6 +957,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 
     fIsBareMultisigStd = GetBoolArg("-permitbaremultisig", true);
     nMaxDatacarrierBytes = GetArg("-datacarriersize", nMaxDatacarrierBytes);
+    nRbfPolicy = GetArg("-replacebyfee", nRbfPolicy, 1);
```

Comment:
Why does not entering a value default to "always", which is not the global default? This is biased towards Full-RBF and not intuitive, because no value for a setting would imply the default value (opt-in) is used.